src/shared/components/event-meta/index.js

```diff
@@ -23,18 +26,20 @@ const EventMeta = ({
           </div>
         : <noscript />
       }
-      {
-        tags
-        ? <TagsList
-            tags={tags}
-            tagsLinkPath="about-us/events" />
-        : <noscript />
-      }
       </div>
     );
   }
 
-  return (<noscript />);
+  if (tags && tags.length > 0) {
```

Comment:
I would put the error handling inside the child component. Easier to unit test that way too.